v2/pkg/core/inputs/hybrid/hmap.go

```diff
@@ -134,3 +144,27 @@ func (i *Input) Scan(callback func(value string)) {
 		i.hostMap.Scan(callbackFunc)
 	}
 }
+
+// expandCIDRInputValue expands CIDR and stores expanded IPs
+func (i *Input) expandCIDRInputValue(value string) {
+	ips, _ := mapcidr.IPAddresses(value)
+	for _, ip := range ips {
+		if _, ok := i.hostMap.Get(ip); ok {
+			i.dupeCount++
+			continue
+		}
+		i.inputCount++
+		_ = i.hostMap.Set(ip, nil)
+		if i.hostMapStream != nil {
+			_ = i.hostMapStream.Set([]byte(ip), nil)
+		}
+	}
+}
+
+// isCIDR returns true if given value is CIDR otherwise false
+func isCIDR(value string) bool {
```

Comment:
I'd suggest removing this and use [`iputil.IsCIDR`](https://github.com/projectdiscovery/iputil/blob/5940e7c42cb040deed0dcda9ed41560f70ce2e16/iputil.go#L74)

Reply:
Removed this function.